### PR TITLE
Adds workload monitoring templates

### PR DESCRIPTION
**Pull request by acearun**
Add workload insights templates for:
1. SQL Azure, SQL Server, NGINX, MySQL, Redis cache
2. Templates for onboarding monitoring.
3. Common analytics - alerting, explore metrics, enable monitoring, troubleshooting and reason monitoring is not supported.

### CODE REVIEW

File: Workbooks/Workloads/Monitoring checklist/Monitoring checklist.workbook [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+            "isHiddenWhenLocked": true,
+            "queryType": 1,
+            "resourceType": "microsoft.resourcegraph/resources"
+          },
+          {
+            "id": "77d59cac-0e77-4a4f-9857-8586aa7ca26e",
+            "version": "KqlParameterItem/1.0",
+            "name": "Text",
+            "type": 1,
+            "query": "Resources\n| take 1\n| project x = dynamic({OnboardingStatus})\n| project label = strcat('**Virtual machine: ', x.Name, '**',\n    '<br /><br />',\n    strcat(iff(x.Location in~ ('centralus', 'eastus', 'westus'), '✅', '❌'), ' Located in ', x.Location),\n    '<br /><br />',\n    strcat(iff(x.OsType =~ 'linux', '✅', '❌'), ' OS is ', x.OsType),\n    '<br /><br />',\n    strcat(iff(x.Offer =~ 'ubuntuserver', '✅', '❌'), ' OS offer is ', x.Offer),\n    '<br /><br />',\n    strcat(iff(x.WorkspaceId != '', '✅', '❌'), ' Configured with the OMS agent'),\n    '<br /><br />',\n    strcat(iff(x.Workspace != '', '✅', '❌'), ' Can access to the Log Analytics workspace')\n    )",
```
Comment (dougbrad) [inline Workbooks/Workloads/Monitoring checklist/Monitoring checklist.workbook:93]: 'Configured with the OMS agent'  
-> OMS was retired long back, change to 'Log Analytics'
' Can access to the Log Analytics workspace' 
-> Not really proper English, also should explain better.

Comment (dougbrad) [inline Workbooks/Workloads/Monitoring checklist/Monitoring checklist.workbook:93]: How will we localize these strings in the query?

Comment (acearun) [reply Workbooks/Workloads/Monitoring checklist/Monitoring checklist.workbook:93]: We don't have a good story for localizating strings coming from queries. We could have it return constants and then use the criteria params with loc'd strings.

Comment (acearun) [reply Workbooks/Workloads/Monitoring checklist/Monitoring checklist.workbook]: Fixed

File: Workbooks/Workloads/SQL Insights/SQL Insights.workbook [hunk 2]
```diff
@@ -0,0 +1,3129 @@
...
+          "{Subscription}"
+        ],
+        "parameters": [
+          {
+            "id": "1f74ed9a-e3ed-498d-bd5b-f68f3836a117",
+            "version": "KqlParameterItem/1.0",
+            "name": "Subscription",
+            "label": "Subscriptions",
+            "type": 6,
+            "description": "All subscriptions with Storage accounts",
```
Comment (dougbrad) [inline Workbooks/Workloads/SQL Insights/SQL Insights.workbook:18]: Description appears to be wrong 'with Storage accounts'

File: Workbooks/Workloads/Collection logs/Collection logs.workbook [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+      "content": {
+        "json": "### Collection logs from {VirtualMachine:name}"
+      },
+      "name": "text - 2"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "telegraflogs_CL\n| order by TimeGenerated desc\n| extend Message = extract(@'.+Z D! \\[outputs.socket_writer\\] (.+)', 1, RawData) \n| project Time = TimeGenerated, Message = iff(Message == '', RawData, Message), ['Log type'] = iff(Type == 'telegraflogs_CL', 'Telegraf', 'WA Agent')\n| take 200",
```
Comment (heyaWorld) [inline Workbooks/Workloads/Collection logs/Collection logs.workbook:45]: We also need logs from "waagentlogs_CL" table. Seems, this query is only getting logs from "telegraflogs_CL"

Comment (acearun) [reply Workbooks/Workloads/Collection logs/Collection logs.workbook:45]: I'll fix this in a separate change.

File: Workbooks/Workloads/Onboard MSSQL Insights/settings.json [hunk 4]
```diff
@@ -0,0 +1,8 @@
+{
+    "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
+    "name":"Onboard MS SQL Insights",
+    "author": "Microsoft",
```
[the remark this answers is not in the document]
Comment (acearun) [reply Workbooks/Workloads/Onboard MSSQL Insights/settings.json:4]: Onboarding templates are in a different gallery, hence don't need the preview tag.

File: Workbooks/Workloads/Alerts group/Alerts group.workbook [hunk 5]
```diff
@@ -0,0 +1,328 @@
...
+        "style": "above",
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces"
+      },
+      "conditionalVisibility": {
+        "parameterName": "1",
+        "comparison": "isEqualTo",
+        "value": "2"
+      },
+      "name": "parameters - 5"
```
[the remark this answers is not in the document]
Comment (acearun) [reply Workbooks/Workloads/Alerts group/Alerts group.workbook]: Renamed

**Pull request merged**